src/UTILS/decompose.jl

```diff
@@ -112,9 +112,9 @@ function CSA_greedy_step(F :: F_OP, do_svd = SVD_for_CSA, print = DECOMPOSITION_
 		end
 	else
 		if do_grad
-			return optimize(cost, grad!, x0, BFGS(), Optim.Options(show_every=print, show_trace=true, extended_trace=true))
```

Comment:
The idea of having the show_every=print option is that we can customize the level of verbose from the config file. If you could leave this with =print instead of =1 it would be great :)